modules/openshift-cluster-limits.adoc

```diff
@@ -13,71 +13,72 @@
 | 2,000
 | 2,000
 | 2,000
-|TBD
+| 250
 
 | Number of pods footnoteref:[numberofpods,The pod count displayed here is the number of test pods. The actual number of pods depends on the application’s memory, CPU, and storage requirements.]
 | 120,000
 | 150,000
 | 150,000
-|TBD
+| 150,000
 
 | Number of pods per node
 | 250
 | 250
 | 250
-|TBD
+| 250
 
 | Number of pods per core
 | 10 is the default value. The maximum supported value is the number of pods per node.
 | There is no default value. The maximum supported value is the number of pods per node.
 | There is no default value. The maximum supported value is the number of pods per node.
-|TBD
+| There is no default value. The maximum supported value is the number of pods per node.
 
-| Number of namespaces
+| Number of namespaces footnoteref:[numberofnamepaces, When there are large number of active projects, etcd may suffer from poor performance if the keyspace grows excessively large and exceeds the space quota. Periodic maintenance of etcd, including defragmentaion, is highly recommended to free etcd storage.]
```

Comment:
I think you missed my first one:   s/When there are large number/When there are a large number/

Reply:
Fixed, thanks.